e2e_tests/tests/fixtures/pytorch_identity/model_def.py

```diff
@@ -0,0 +1,64 @@
+from typing import Any, Dict, Tuple
+
+import torch.utils.data
+
+from determined import pytorch
+
+
+class IdentityDataset(torch.utils.data.Dataset):
+    def __init__(self, initial_value: int = 0):
+        self.initial_value = initial_value
+
+    def __len__(self) -> int:
+        return 64
+
+    def __getitem__(self, index: int) -> Tuple:
+        v = 1  # float(self.initial_value + index)
```

Comment:
I'm thinking that this test does not inspire enough confidence given that every batch is going to have the same data.

Can you make it a triangle dataset, like what you have in the comment?  Probably the slope of the dataset needs to be less than one or your SGD will explode (meaning, `self.initial_value + 0.001 * index` or something)